jenkins/pipelines/continuous-delivery/Jenkinsfile

```diff
@@ -7,6 +7,13 @@ node('COMPONENT_ECS') {
                 scp -o StrictHostKeyChecking=no web/target/*.war ${DEV_USER}@${DEV_HOST}:/testgrid/deployment/apache-tomcat-8.5.23/webapps/ROOT.war
                 scp -o StrictHostKeyChecking=no deployment-tinkerer/target/*.war ${DEV_USER}@${DEV_HOST}:/testgrid/deployment/apache-tomcat-8.5.23/webapps/
                 scp -o StrictHostKeyChecking=no distribution/target/*.zip ${DEV_USER}@${DEV_HOST}:/testgrid/testgrid-home/testgrid-dist/WSO2-TestGrid.zip
+                ssh -o StrictHostKeyChecking=no ${DEV_USER}@${DEV_HOST} /bin/bash << EOF
+                    cd /testgrid/testgrid-home/testgrid-dist/;
+                    unzip WSO2-TestGrid.zip -d WSO2-TestGrid-unzipped;
```

Comment:
If we are unzipping the distribution from the CD pipeline we should remove the unzip step from the Job pipeline WDYT?

Reply:
+1.. I removed it.